llvm/test/CodeGen/Xtensa/bswap.ll

```diff
@@ -0,0 +1,413 @@
+; RUN: llc -mtriple=xtensa -verify-machineinstrs < %s \
```

Comment:
All the tests are missing "autogenerated by" annotation.

Reply:
Fixed